tensorflow/lite/micro/examples/person_detection/esp/app_camera_esp.h

```diff
@@ -0,0 +1,183 @@
+#define VSYNC_GPIO_NUM   25
+#define HREF_GPIO_NUM    23
+#define PCLK_GPIO_NUM    22
+
+#elif CONFIG_CAMERA_MODEL_ESP_EYE
+#define PWDN_GPIO_NUM    -1
+#define RESET_GPIO_NUM   -1
+#define XCLK_GPIO_NUM    4
+#define SIOD_GPIO_NUM    18
+#define SIOC_GPIO_NUM    23
+
+#define Y9_GPIO_NUM      36
+#define Y8_GPIO_NUM      37
+#define Y7_GPIO_NUM      38
+#define Y6_GPIO_NUM      39
+#define Y5_GPIO_NUM      35
+#define Y4_GPIO_NUM      14
+#define Y3_GPIO_NUM      13
+#define Y2_GPIO_NUM      34
+#define VSYNC_GPIO_NUM   5
+#define HREF_GPIO_NUM    27
+#define PCLK_GPIO_NUM    25
+
+#elif CONFIG_CAMERA_MODEL_M5STACK_PSRAM
+#define PWDN_GPIO_NUM     -1
+#define RESET_GPIO_NUM    15
+#define XCLK_GPIO_NUM     27
+#define SIOD_GPIO_NUM     25
+#define SIOC_GPIO_NUM     23
+
+#define Y9_GPIO_NUM       19
+#define Y8_GPIO_NUM       36
+#define Y7_GPIO_NUM       18
+#define Y6_GPIO_NUM       39
+#define Y5_GPIO_NUM        5
+#define Y4_GPIO_NUM       34
+#define Y3_GPIO_NUM       35
+#define Y2_GPIO_NUM       32
+#define VSYNC_GPIO_NUM    22
+#define HREF_GPIO_NUM     26
+#define PCLK_GPIO_NUM     21
+
+#elif CONFIG_CAMERA_MODEL_M5STACK_WIDE
+#define PWDN_GPIO_NUM     -1
+#define RESET_GPIO_NUM    15
+#define XCLK_GPIO_NUM     27
+#define SIOD_GPIO_NUM     22
+#define SIOC_GPIO_NUM     23
+
+#define Y9_GPIO_NUM       19
+#define Y8_GPIO_NUM       36
+#define Y7_GPIO_NUM       18
+#define Y6_GPIO_NUM       39
+#define Y5_GPIO_NUM        5
+#define Y4_GPIO_NUM       34
+#define Y3_GPIO_NUM       35
+#define Y2_GPIO_NUM       32
+#define VSYNC_GPIO_NUM    25
+#define HREF_GPIO_NUM     26
+#define PCLK_GPIO_NUM     21
+
+#elif CONFIG_CAMERA_MODEL_AI_THINKER
+#define PWDN_GPIO_NUM     32
+#define RESET_GPIO_NUM    -1
+#define XCLK_GPIO_NUM      0
+#define SIOD_GPIO_NUM     26
+#define SIOC_GPIO_NUM     27
+
+#define Y9_GPIO_NUM       35
+#define Y8_GPIO_NUM       34
+#define Y7_GPIO_NUM       39
+#define Y6_GPIO_NUM       36
+#define Y5_GPIO_NUM       21
+#define Y4_GPIO_NUM       19
+#define Y3_GPIO_NUM       18
+#define Y2_GPIO_NUM        5
+#define VSYNC_GPIO_NUM    25
+#define HREF_GPIO_NUM     23
+#define PCLK_GPIO_NUM     22
+
+
+#elif CONFIG_CAMERA_MODEL_CUSTOM
+#define PWDN_GPIO_NUM    CONFIG_CAMERA_PIN_PWDN
+#define RESET_GPIO_NUM   CONFIG_CAMERA_PIN_RESET
+#define XCLK_GPIO_NUM    CONFIG_CAMERA_PIN_XCLK
+#define SIOD_GPIO_NUM    CONFIG_CAMERA_PIN_SIOD
+#define SIOC_GPIO_NUM    CONFIG_CAMERA_PIN_SIOC
+
+#define Y9_GPIO_NUM      CONFIG_CAMERA_PIN_Y9
+#define Y8_GPIO_NUM      CONFIG_CAMERA_PIN_Y8
+#define Y7_GPIO_NUM      CONFIG_CAMERA_PIN_Y7
+#define Y6_GPIO_NUM      CONFIG_CAMERA_PIN_Y6
+#define Y5_GPIO_NUM      CONFIG_CAMERA_PIN_Y5
+#define Y4_GPIO_NUM      CONFIG_CAMERA_PIN_Y4
+#define Y3_GPIO_NUM      CONFIG_CAMERA_PIN_Y3
+#define Y2_GPIO_NUM      CONFIG_CAMERA_PIN_Y2
+#define VSYNC_GPIO_NUM   CONFIG_CAMERA_PIN_VSYNC
+#define HREF_GPIO_NUM    CONFIG_CAMERA_PIN_HREF
+#define PCLK_GPIO_NUM    CONFIG_CAMERA_PIN_PCLK
+#endif
+
```

Comment:
In the case when no known camera model is defined (I believe this can happen if the user forgot to configure), no obvious message will be shown, only some missing symbols errors.

Can you please add an `#else` clause at the end with an error message using the `#error` macro ? (Something like `#error Camera model is not configured`).

Reply:
@fredrec thank you for reviewing, Actually the default module is WROVER_MODULE, so it will be always selected in `Kconfig`. but anyways, I will add the necessary changes to make it foolproof.